cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -689,6 +692,19 @@ private RuntimeJobStatus getRuntimeJobStatus(Job job) {
     return runtimeJobStatus;
   }
 
+  /**
+   * Returns job state details, such as an error description if the state is ERROR.
+   * For other job states, returns null.
+   */
+  @Nullable
+  private RuntimeJobStatusDetails getJobStatusDetails(Job job) {
```

Comment:
Is the new JobStatusDetails string logged somewhere if there is a failure during execution? I looked at RuntimeJobManager.getDetail, but it was not getting logged.

Reply:
It is logged here: https://github.com/cdapio/cdap/blob/629e699112509d2e904d6f4232b3d2f8dd93589a/cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/AbstractDataprocProvisioner.java#L124

Reply:
The `RuntimeJobStatusDetails` is a parameterized type. Please don't use it without the parameterized type. You should see a warning in IDE for usage like this

Reply:
Fixed.